Bottom: 24fc463d12c5e6f6b2a878211d4ee56eb1d2c08c
Top:    17566a5ed8e0a7941077a8c3dc4259304f96c9bd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 20:26:14 +0000

nelma: Fix prototype on set_layer() method to include the "..." flag.



---

diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index b27f8e9..964265a 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -698,7 +698,7 @@ nelma_do_export(HID_Attr_Val * options)
 /* *** PNG export (slightly modified code from PNG export HID) ************* */
 
 static int 
-nelma_set_layer(const char *name, int group)
+nelma_set_layer(const char *name, int group, int empty)
 {
 	int             idx = (group >= 0 && group < max_layer) ?
 	PCB->LayerGroups.Entries[group][0] : group;
